src/main/java/teammates/ui/output/NotificationData.java

```diff
@@ -73,10 +73,19 @@ public boolean isShown() {
         return this.shown;
     }
 
+    public void setCreatedAt(long timestamp) {
+        this.createdAt = timestamp;
+    }
+
+    public void setTargetUser(NotificationTargetUser user) {
+        this.targetUser = user;
+    }
+
     /**
      * Hides some attributes to instructor and students without appropriate privilege.
      */
     public void hideInformationForNonAdmin() {
-        // TODO: hide information that might be admin-only: target users, isShown, (starttime and endtime)
+        setCreatedAt(0);
+        setTargetUser(null);
```

Comment:
Any reason for hiding these? It seems to be ok for them to be public information.

Reply:
I think it might be good to hide information that non-admin does not use, but as you said these might not be necessary. I will remove those hiding in the next commit.